.circleci/config.yml

```diff
@@ -0,0 +1,164 @@
+version: 2
+
+jobs:
+  checkout_code:
+    docker:
+      - image: jfullaondo/datadog-agent-builder-deb:0.0.2
+    working_directory: /go/src/github.com/DataDog/datadog-agent
+    steps:
+      - checkout
+      - run:
+          name: save SHA to a file
+          command: echo $CIRCLE_SHA1 > .circle-sha
+      - save_cache:
+          key: v1-repo-{{ checksum ".circle-sha" }}
+          paths:
+            - /go/src/github.com/DataDog/datadog-agent
+
+  dependencies:
+    docker:
+      - image: jfullaondo/datadog-agent-builder-deb:0.0.2
+    working_directory: /go/src/github.com/DataDog/datadog-agent
+    steps:
+      - run:
+          name: save SHA to a file
+          command: echo $CIRCLE_SHA1 > .circle-sha
+      - restore_cache:
+          keys:
+            - v1-repo-{{ checksum ".circle-sha" }}
+      - run:
+          name: grab go deps
+          command: invoke deps
+      - save_cache:
+          key: v1-govendordeps-{{ checksum "Gopkg.lock" }}
```

Comment:
Would it make sense to reuse that cache if `Gopkg.lock` is the same between PRs, instead of running `inv deps` everytime? Is that possible?

Reply:
I think you can achieve this by adding the `v1-govendordeps-{{ checksum "Gopkg.lock" }}` in the `restore_cache` step above